{
  "contest": 261,
  "handle": "wisdomn_",
  "risk": "3",
  "title": "Function emergencyWithdraw is vulnerable to reentrancy (Severe)",
  "issueId": 465,
  "issueUrl": "https://github.com/code-423n4/2023-07-tapioca-findings/issues/465"
}